portal-frontend/src/app/features/notice-of-intents/alcs-edit-submission/alcs-edit-submission.component.scss

```diff
@@ -194,11 +194,15 @@
     }
 
     .button-container {
```

Comment:
Suggestion: add hack to match Figma:
```css
box-shadow: -3px 0 colors.$white, 3px 0 colors.$white;
```

If you can think of a better way to do this, let me know.